### DISCUSSION TITLE
Added oauth - certificate is not recognised anymore

### DISCUSSION THREAD

**Discussion by theluke** [htpcBeginner/docker-traefik#226] [Support]
Hi all,

Still working out the kinks on my implementation.

I am working on a Ubuntu Server 20.04

I followed the steps to add **oauth**, and it worked, since I am prompted for my 2FA Google Credentials... _but_ Chrome does not connect anymore to the Traefik V2 Dashboard since it does not recognise the certificate.

Let's take a look at the logs:

```
> 2022-01-05T15:29:03.592651107Z time="2022-01-05T15:29:03Z" level=error msg="middleware \"chain-oauth@file\" does not exist" routerName=traefik-rtr@docker entryPointName=https
> 2022-01-05T15:36:24.765009492Z time="2022-01-05T15:36:24Z" level=error msg="The ACME resolver \"dns-cloudflare\" is skipped from the resolvers list because: unable to get ACME account: read /acme.json: is a directory"
> 2022-01-05T15:36:43.912626807Z time="2022-01-05T15:36:43Z" level=debug msg="http: TLS handshake error from 109.xxx.xxx.xxx::52842: remote error: tls: unknown certificate"
> 2022-01-05T15:37:49.601465962Z time="2022-01-05T15:37:49Z" level=debug msg="Remote error http://oauth:4181. StatusCode: 307" middlewareType=ForwardedAuthType middlewareName=middlewares-oauth@file
> 2022-01-05T15:43:14.431403027Z time="2022-01-05T15:43:14Z" level=debug msg="http: panic serving 109.xxx.xxx.xxx::53006: runtime error: invalid memory address or nil pointer dereference"

```

Specifically, this error goes on and on:
```
level=debug msg="http: TLS handshake error from 109.xxx.xxx.xxx:53117: remote error: tls: unknown certificate
```

It was working fine and the certificate was recognised before turning on **oauth**.

Appreciate any help to hunt down these bugs!

**Comment by theluke** [comment]
I **think** this has something to do with my error, but I do not understand what should I change:

https://community.traefik.io/t/tls-unknown-certificate/2687

Also traefik/traefik-helm-chart#164

**Comment by SimpleHomelab** [comment]
Traefik is unable to find "chain-oauth" middleware. Check your middlewares-chains.yml (or toml).
